cico_run_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# We need to disable selinux for now, XXX
+/usr/sbin/setenforce 0
+
+# Get all the deps in
+yum -y install docker make \
+       git golang 
```

Comment:
We don't need golang on the worker as it is installed with the `Dockerfile.builder` already.